subgraphs/aave-forks/schema.graphql

```diff
@@ -1,5 +1,5 @@
-# Subgraph Schema: Lending Protocol (Extended)
-# Version: 2.0.1
+# Subgraph Schema: Lending Protocol
+# Version: 3.0.2
```

Comment:
Let's not update the version since the additions are just new enums.

Could you add the new enum fields to the overall schema and add them to the SDK reference for lending?

Reply:
done.